fastlane/metadata/android/en-US/changelogs/983.txt

```diff
@@ -0,0 +1,9 @@
+New fast forward / rewind behaviour
+Made settings searchable
+Highlight pinned comments as such
+Added FSFE PeerTube instance to URL opener
+Improved error reporting and new error notification
```

Comment:
The error notification has been added and not improved, so I think we should reflect (properly) this in the changelog:

```suggestion
Improved error reporting and add new error notification
```